pom.xml

```
@@ -76,7 +76,7 @@
 
     <properties>
         <version.arquillian_core>1.7.0.Alpha10</version.arquillian_core>
-        <version.jersey>3.0.1</version.jersey>
+        <version.jersey>3.0.4</version.jersey>
         <version.jakartaee>9</version.jakartaee>
         <version.jakartaee.api>9.1.0-RC1</version.jakartaee.api>
```

Can probably bump this while we're at it

```suggestion
        <version.jakartaee.api>9.1.0</version.jakartaee.api>
```